### DISCUSSION TITLE
Docs get blocked often

### DISCUSSION THREAD

**Discussion by nandorojo** [dotansimha/graphql-code-generator#8549] [General] [answered]
<img width="1770" alt="199071552-e368fad0-fa4b-4b5e-8b3f-b0eeb16a89ac" src="https://user-images.githubusercontent.com/13172299/199071910-fc23c008-191a-4a93-8958-ce55004385fd.png">

I get this error almost every time I try to open the docs. If I refresh it often shows an unstyled page. When I use incognito it seems to work fine. I'm in NYC so I'm not sure why this would be the case. Thought I'd flag it. I don't get this on any other sites.

**Comment by dotansimha** [comment, marked as answer]
Hi @nandorojo , sorry for the inconvenience. We recently worked a lot to improve our documentation websites, and also improved some security aspects. We noticed some false positives on the firewall. 
I did some adjustments, can you please try again now?

**Reply by nandorojo** [reply]
I think it's fixed, thanks!
